testsuite/integration/basic/pom.xml

```diff
@@ -1284,6 +1323,39 @@
                                     </includes>
                                 </configuration>
                             </execution>
+                            <!-- Tests against the install with batch-jberet -->
+                            <execution>
+                                <id>batch-jberet-layers.surefire</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <jboss.install.dir>${basedir}/target/wildfly-batch</jboss.install.dir>
+                                        <jboss.home>${project.build.directory}/wildfly-batch</jboss.home>
+                                        <jboss.home.dir>${project.build.directory}/wildfly-batch</jboss.home.dir>
+                                        <jbossas.dist>${project.build.directory}/wildfly-batch</jbossas.dist>
+                                        <jboss.dist>${project.build.directory}/wildfly-batch</jboss.dist>
+                                        <!-- Override the standard module path that points at the shared module set from dist -->
+                                        <module.path>${project.build.directory}/wildfly-batch/modules${path.separator}${basedir}/target/modules</module.path>
+                                    </systemPropertyVariables>
+                                    <environmentVariables>
+                                        <JBOSS_HOME>${project.build.directory}/wildfly-batch</JBOSS_HOME>
+                                    </environmentVariables>
+                                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                                    <includes>
+                                        <include>org/jboss/as/test/integration/batch/**/*TestCase.java</include>
+                                    </includes>
+                                    <excludes>
+                                        <!-- Test app has a manifest dep on org.jboss.remoting3 module -->
```

Comment:
From the recent discussions taking about this, we should be able to replace these org.jboss.remoting3 by a non-deprecated module. @bstansberry do you want me to driving this PR instead of you? I can create another one to supersedes this.